### PR TITLE
Meeting notes 15 june 2016

### PR DESCRIPTION
**Pull request by hjdivad**

### CODE REVIEW

**Comment by hjdivad** [comment]
I'll update the PR with notes from the meeting but in the meantime, can list agenda here

File: ember-cli/2017-06/june-15.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# ember-cli Meeting Notes - 2017-06-15
+
+## Attendees
+
+
+## Agenda
+
```
Comment (Turbo87) [inline ember-cli/2017-06/june-15.md]: ember-cli-shims v1.2.0

Comment (rwjblue) [inline ember-cli/2017-06/june-15.md]: * Testing
  * deprecate usage of restricted resolver - https://github.com/emberjs/rfcs/pull/229
  * new testing API - https://github.com/emberjs/rfcs/pull/232

Comment (kratiahuja) [inline ember-cli/2017-06/june-15.md]: ember-cli-fastboot@rc.4 after fingerprinting is merged

Comment (stefanpenner) [inline ember-cli/2017-06/june-15.md]: - npm v5 / node 8
- shutdown code

Comment (kellyselden) [inline ember-cli/2017-06/june-15.md]: hydration updates?

**Comment by hjdivad** [comment]
cc 
@kratiahuja
@rwjblue 
@stefanpenner
@turbo87 
@kellyselden

**Comment by hjdivad** [comment]
and obligatory https://www.youtube.com/watch?v=jNKjShmHw7s

**Comment by kratiahuja** [comment]
LGTM!

**Pull request merged**